src/LitePatch/LitePatch.Services/Interfaces/ISettingsService.cs

```diff
@@ -1,10 +1,13 @@
 ﻿using LitePatch.Services.Models;
+using LitePatch.Services.Repo;
 
 namespace LitePatch.Services.Interfaces;
 
 public interface ISettingsService
 {
+    public SettingsRepository SettingsRepository { get; set; }
```

Comment:
This can also be removed, since we do not need to access the repository from outside the service.

Reply:
Has been amended